packages/components/Tabs/src/Tabs.settings.ts

```diff
@@ -0,0 +1,27 @@
+import { IComposeSettings } from '@uifabricshared/foundation-compose';
+import { TabsType, tabsName } from './Tabs.types';
+
+export const settings: IComposeSettings<TabsType> = [
+  {
+    tokens: {
+      color: 'menuItemText',
+    },
+    root: {
+      accessible: true,
+      accessibilityRole: 'tablist',
+    },
+    label: {
+      style: {
+        fontFamily: 'inherit',
```

Comment:
I would remove this - it recently caused an issue with RadioGroup on MacOS and I don't think it's necessary here